addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -45,18 +45,26 @@ def _default_user_ids(self):
         compute='_are_you_inside', store=False, readonly=True)
     script_external = fields.Html('Script (external)', compute='_compute_script_external', store=False, readonly=True, sanitize=False)
     nbr_channel = fields.Integer('Number of conversation', compute='_compute_nbr_channel', store=False, readonly=True)
+    chatbot_count = fields.Integer(string='Number of Chatbot', compute='_compute_chatbot_ids', store=True)
 
     image_128 = fields.Image("Image", max_width=128, max_height=128, default=_default_image)
 
     # relationnal fields
     user_ids = fields.Many2many('res.users', 'im_livechat_channel_im_user', 'channel_id', 'user_id', string='Operators', default=_default_user_ids)
     channel_ids = fields.One2many('mail.channel', 'livechat_channel_id', 'Sessions')
+    chatbot_ids = fields.Many2many('im_livechat.chatbot.script', compute='_compute_chatbot_ids', store=True)
     rule_ids = fields.One2many('im_livechat.channel.rule', 'channel_id', 'Rules')
 
     def _are_you_inside(self):
         for channel in self:
             channel.are_you_inside = bool(self.env.uid in [u.id for u in channel.user_ids])
 
+    @api.depends('rule_ids.chatbot_id')
+    def _compute_chatbot_ids(self):
+        for record in self:
+            record.chatbot_ids = record.rule_ids.mapped('chatbot_id')
+            record.chatbot_count = len(record.chatbot_ids)
```

Comment:
A bit strange to store those 2, seems like duplicated info just for a stat button?